### PR TITLE
Feat/met 5352 sandbox tier mockup interaction improvements 2

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

**Review by jeortizquan — COMMENTED** [review]
Check the comments if they apply.

File: projects/sandbox/src/app/_mocked/mocked-sandbox.service.ts [hunk 1]
```diff
@@ -293,7 +294,7 @@ export class MockSandboxServiceErrors extends MockSandboxService {
 // Temporary code for tier generation
 const varAlphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
 const valStringMetadata = varAlphabet.substr(0, 4);
-const licenses = ['CC0', 'CC1', 'In Copyright', 'CC-BY-SA', 'CC-BY-SA-NC', 'CC-BY'];
+const licenses = [LicenseType.OPEN, LicenseType.CLOSED, LicenseType.RESTRICTED];
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/app/_mocked/mocked-sandbox.service.ts:297]: It was requested explicitly.

File: projects/sandbox/src/assets/sass/themes/theme-white/_theme-white.scss [hunk 2]
```diff
@@ -43,14 +43,13 @@ select {
...
-  background: $grey-3;
+  background-color: $grey-3;
 }
 
 .tier-data-grid {
   .summary-value {
     color: $white;
-    background: $grey-3;
-    background: #aaa;
+    background-color: #aaa;
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/assets/sass/themes/theme-white/_theme-white.scss:52]: It isn't reused!

File: projects/sandbox/src/app/progress-tracker/progress-tracker.component.spec.ts [hunk 3]
```diff
@@ -78,6 +80,26 @@ describe('ProgressTrackerComponent', () => {
       expect(component.getLabelClass('' as StepStatus)).toEqual('harvest');
     });
 
+    it('should prompt a tier data load', () => {
+      const completedDataset: Dataset = Object.assign({}, mockDataset) as Dataset;
+      completedDataset.status = DatasetStatus.COMPLETED;
+      //component.setActiveSubSection(DisplayedSubsection.TIERS);
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/app/progress-tracker/progress-tracker.component.spec.ts]: Done!

**Review by jeortizquan — APPROVED** [review]

**Pull request merged**